scripts/validate_pipelines.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+# Standard
+import glob
+import json
+import sys
+
+# Third Party
+from jsonschema import validate
+import jsonschema
+import yaml
+
+
+def validate_yaml_file(yaml_file, schema):
+    with open(yaml_file, "r") as file:
+        pipeline = yaml.safe_load(file)
+
+    try:
+        validate(instance=pipeline, schema=schema)
+        print(f"Validation successful for {yaml_file}.")
+    except jsonschema.exceptions.ValidationError as err:
+        print(f"Validation failed for {yaml_file}:", err)
+        return False
+    return True
+
+
+def main():
+    schema_path = "src/instructlab/sdg/pipelines/schema/v1.json"
```

Comment:
How about loading this from the `instructlab.sdg` package

Reply:
yes, that would be better